hack/jenkins/upload-finished.sh

```diff
@@ -29,9 +29,17 @@ if [[ $# -ne 1 ]]; then
     exit 1
 fi
 
+# TODO: DRY. Refactor into upload-to-gcs.sh ?
+: ${JENKINS_GCS_LOGS_PATH:="gs://kubernetes-jenkins/logs"}
+: ${JENKINS_UPLOAD_TO_GCS:="y"}
+
+if [[ ! ${JENKINS_UPLOAD_TO_GCS:-} =~ ^[yY]$ ]]; then
```

Comment:
It's a copy pasta from hack/jenkins/upload-to-gcs.sh  Personally I'd rather keep it so that we can eventually refactor all the upload scripts with minimal mental effort.  But I don't really care, so I can change it if you'd like...